### ISSUE TITLE
[FEATURE] Manual Sync Correction

### ISSUE THREAD

**Issue by Coasternaut** [robvdpol/RaceControl#29]
This would basically help overcome some the of issues of the auto sync since it's still being improved. This would allow you to type in the time that a feed is delayed which would then be taken into account when the sync button is pressed. For example, I notice that one feed is delayed by 5 seconds to the rest of them, I could enter 5 seconds into a box on that player and it would then set that feed ahead by 5 seconds of what the sync estimation is when I press sync. This would help when skipping through a race so you don't have to manually fix the sync by pausing all the other feeds.

**Comment by robvdpol** [comment]
I'll have a look to see if this is possible.

**Comment by politbureau** [comment]
I agree with this 100% - the ability to adjust each stream << & >> in the transport control area would be great. Alternatively a sync control panel where you can adjust the sync of each open stream +/- by 'nudging' backwards and forwards, or by entering a manual offset in ms. In addition to this it would be great to be able to save these sync settings to a json or ini file so they can be reloaded when opening the same race again. This would be great as users could share their ini files on the web for watching perfectly synced past races.

**Comment by QuentinPi** [comment]
I'm also interested by this feature for live feeds and have a global pause button.

**Comment by Triodes** [comment]
@QuentinPi There is a keyboard shortcut for pause all: Shift + Space

**Comment by JCvanDoorn** [comment]
I would also be interested in being able to delay all streams by a manual input for the number of seconds. The reason for this is that Live TV Broadcasts usually are about 10 to 15 seconds behind the direct feed through F1 TV Pro. This way, I see an accident on the Stream first, before I see it on the live TV Feed with local commentary. It would be such a big improvement if we can manually adjust a delay so that the TV and the second screen (this app) are nearly synchronised.

**Comment by RaJeronimo** [comment]
This would be a very nice feature.
